### PR TITLE
Configure uwsgi

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Nginx configuration part for https://github.com/mozilla/addons-server/issues/6668

Unfortunately this breaks `make debug` for now afaik which I think can be tackled in a separate step. Fixing cpu load related issues from #11539 seems more important.

### CODE REVIEW

File: addons.conf [hunk 1]
```diff
@@ -48,21 +44,19 @@ server {
...
+        uwsgi_read_timeout 60;
+        uwsgi_connect_timeout 60;
 
-        proxy_connect_timeout 30;
-        proxy_read_timeout 30;
+        uwsgi_param   Host                  $http_host;
+        uwsgi_param   X-Real-IP             $remote_addr;
+        uwsgi_param   X-Forwarded-For       $proxy_add_x_forwarded_for;
+        uwsgi_param   X-Forwarded-Protocol  ssl;
+        uwsgi_param   Vary                  User-Agent;
```
Comment (bqbn) [inline addons.conf]: Production passes more params on, like these,

```
uwsgi_param HTTPS on;

uwsgi_param  QUERY_STRING       $query_string;
uwsgi_param  REQUEST_METHOD     $request_method;
uwsgi_param  CONTENT_TYPE       $content_type;
uwsgi_param  CONTENT_LENGTH     $content_length;

uwsgi_param  REQUEST_URI        $request_uri;
uwsgi_param  PATH_INFO          $document_uri;
uwsgi_param  DOCUMENT_ROOT      $document_root;
uwsgi_param  SERVER_PROTOCOL    $server_protocol;

uwsgi_param  REMOTE_ADDR        $remote_addr;
uwsgi_param  REMOTE_PORT        $remote_port;
uwsgi_param  SERVER_PORT        $server_port;
uwsgi_param  SERVER_NAME        $server_name;
```

Not sure locally the app needs all those or not.

Comment (bqbn) [inline addons.conf]: Also in production, we don't pass the "Vary" header to the app, we instead `add_header Vary User-Agent;`, which basically add a "Vary" header when Nginx replies to the request.


You can see https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/amo/puppet/modules/amo_proxy/templates/nginx.addons.conf.erb#L516-L533 for what we do for the addons-server.

**Review by bqbn — APPROVED** [review]
r+wc

File: addons.conf (hunk 1 above)
Comment (EnTeQuAk) [reply addons.conf]: Most of these params are passed on via `include uwsgi_params` and defined there (https://github.com/nginx/nginx/blob/master/conf/uwsgi_params). So I think this should be good.

Comment (EnTeQuAk) [reply addons.conf]: Indeed, fixed in 7f2473d

**Comment by EnTeQuAk** [comment]
I pushed this branch with the tag `uwsgi` to docker-hub to test this easier.

**Pull request merged**